bundles/org.openhab.binding.fronius/src/main/java/org/openhab/binding/fronius/internal/FroniusBindingConstants.java

```diff
@@ -47,6 +48,7 @@
     public static final String PowerFlowpGrid = "powerflowchannelpgrid";
     public static final String PowerFlowpLoad = "powerflowchannelpload";
     public static final String PowerFlowpAkku = "powerflowchannelpakku";
+    public static final String PowerFlowpPv = "powerflowchannelppv";
```

Comment:
Actually constants should be all upper case. Would be awesome if you could fix all in this class.
```suggestion
    public static final String POWER_FLOW_P_PV = "powerflowchannelppv";
```